.github/workflows/ci.yml

```diff
@@ -1,9 +1,7 @@
 name: CI Checks
 on:
   push:
-    branches: ["**"]
   pull_request:
-    branches: ["main"]
```

Comment:
Seems like the lines were just removed?

Reply:
yeah, without that, it would act like **

Reply:
'**' | Matches all branch and tag names. This is the default behavior when you don't use a branches or tags filter.
-- | --


https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet